system/mp/mpcomm.cpp

```diff
@@ -2677,10 +2678,13 @@ CMPServer::CMPServer(unsigned __int64 _role, unsigned _port, bool _listen)
 CMPServer::~CMPServer()
 {
 #ifdef _TRACEORPHANS
-    StringBuffer buf;
-    getReceiveQueueDetails(buf);
-    if (buf.length())
-        LOG(MCdebugInfo, "MP: Orphan check\n%s",buf.str());
+    if (dumpQueue)
```

Comment:
is this really an incidental change? Why suppress dumping of unconsumed MP messages if slave has been told to shutdown by manager, vs if it hasn't?

Reply:
On clean shutdown I didn't want the MP Orphan check dump in the logs

Reply:
ok, I don't think message are useful in general.